packages/gatsby-source-drupal/src/normalize.ts

```diff
@@ -154,7 +158,12 @@ export const nodeFromData = async (
     typeof attributeId !== `undefined` ? { _attributes_id: attributeId } : {}
 
   const langcode = attributes.langcode || `und`
-  const type = datum.type.replace(/-|__|:|\.|\s/g, `_`)
+  const { typePrefix = `` } = pluginOptions
```

Comment:
It might be good to ensure type prefix is pascal case. I ran into a couple problems with contentful in the past where either graphql-compose or something internal in Gatsby (can't remember which) pascal cased a type but contentful camel cased it and it led to weird bugs. 
So in this case if the user lower cases the first letter of the prefix it may lead to unexpected results in the future.

Reply:
Good idea!